src/classes/Model/Users/InsertToken.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+namespace dhope0000\LXDClient\Model\Users;
+
+use dhope0000\LXDClient\Model\Database\Database;
+
+class InsertToken
+{
+    public function __construct(Database $database)
+    {
+        $this->database = $database->dbObject;
+    }
+
+    public function insert(string $token, string $userId)
```

Comment:
small thing but can we do 
```suggestion
    public function insert(string $userId, string $token) 
```

Reply:
Done.

Reply:
Done.